src/mudlet.cpp

```diff
@@ -130,6 +130,10 @@ bool TConsoleMonitor::eventFilter(QObject* obj, QEvent* event)
 
 mudlet::mudlet()
 : QMainWindow()
+{
```

Comment:
Brilliant solution! I love how clean the diff is.

Could you add a comment here explaining the "why" of having an empty constructor is a good idea?